helm/alfresco-content-services/values.yaml

```diff
@@ -538,6 +551,70 @@ elasticsearch:
   enabled: true
   replicas: 1
   clusterHealthCheckParams: "wait_for_status=yellow&timeout=1s"
+elasticsearchAudit:
+  # -- Enables the embedded elasticsearch cluster for alfresco-audit-storage
+  enabled: true
+  nameOverride: elasticsearch-aas
+  replicas: 1
+  clusterHealthCheckParams: "wait_for_status=yellow&timeout=1s"
+  clusterName: elasticsearch-aas
+  ingress:
+    enabled: true
+    annotations: 
+      nginx.ingress.kubernetes.io/rewrite-target: /$2
+    hosts:
+      - paths:
+          - path: /elasticsearch(/|$)(.*)
+            pathType: Prefix
```

Comment:
we can't expose elasticsearch via ingress directly as it doesn't have any authentication, is that mandatory for kibana?

Reply:
It's correct. The audit storage doesn't have any UI/API and we need to access the ES from tests to verify everything works correctly. @cezary-witkowski created a test env where ES is protected with basic auth and the password is taken from the pipeline secrets.

Reply:
for sure we can't have here a default configuration which is insecure, it's fine that we enabled it on internal environments behind basic auth

Reply:
otherwise if it's needed just for tests let's have ES not exposed , and components coudl use the service directly, and we add the ingress only in TF pipeline?